schema/index.js

```diff
@@ -101,7 +102,8 @@ const schema = new GraphQLSchema({
     fields: {
       followArtist: FollowArtist,
       updateCollectorProfile: UpdateCollectorProfile,
-      updateConversation: UpdateConversation,
+      updateBuyerOutcome: UpdateBuyerOutcome,
+      appendConversationThread: AppendConversationThread,
```

Comment:
I don't think its a big deal whether to encapsulate these into more generically-named ones via some input parameters, or keep it specifically named like here.

If the conversation one is to be renamed, we have to update `reaction-force`, and hopefully do that together so as not to be out of sync too long/or forget.